cordova-lib/src/cordova/util.js

```diff
@@ -262,3 +270,81 @@ function addModuleProperty(module, symbol, modulePath, opt_wrap, opt_obj) {
         });
     }
 }
+
+function resolvePath(pPath) {
```

Comment:
Fixed in: https://github.com/MSOpenTech/cordova-lib/commit/9d267df1b389ef1f70dcbca4292e1ffd6641e895